### DISCUSSION TITLE
Pura login when using Google?

### DISCUSSION THREAD

**Discussion by derekcentrico** [natekspencer/hacs-pura#63] [Q&A] [answered]
The script requires login via user/pass.  However, I used Google for my account for Pura.  How does one get a user/pass setup through Pura for this purpose?  I can't figure that out when toying on their site.  Maybe I'm just blind. ;-D

**Comment by natekspencer** [comment, marked as answer]
Someone mentioned that you can use the "forgot your password" feature with your google email address to then set a password for your Pura account: https://github.com/natekspencer/hacs-pura/issues/40#issuecomment-1868387854

**Comment by cummingsk05** [comment]
Thank you!
